### PR TITLE
LET _parse_txt routine

### PR DESCRIPTION
**Pull request by ankitkmr**
This function is intended to read in data from LET data file stored on hard disk and return the header as a list and data as pandas dataframe. I'll implement the class structure around it soon.

### CODE REVIEW

File: sunpy/lightcurve/sources/let.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    Parses a STEREO LET file from
+    http://www.srl.caltech.edu/STEREO/Public/LET_public.html
+
+    """
+    #Reading in Header  
+
+    type_of_data = ''
+    header = []
+    flag1 = -1
+    flag2 = -1
```
Comment (dpshelio) [inline sunpy/lightcurve/sources/let.py]: I would not use `flags` but a dictionary (if they refer to more than one thing) and `True` or `False` to test them

**Comment by ehsteve** [comment]
This belongs in a STEREO module not its own module.

**Comment by ehsteve** [comment]
Also PRs require a description.

File: sunpy/lightcurve/sources/let.py (hunk 1 above)
Comment (ankitkmr) [inline sunpy/lightcurve/sources/let.py]: made all the changes :)

**Pull request closed without merging**